dotnet/src/Functions/Functions.OpenApi/Model/RestApiOperationRunOptions.cs

```diff
@@ -18,4 +18,19 @@ internal sealed class RestApiOperationRunOptions
     /// The URL of REST API host.
     /// </summary>
     public Uri? ApiHostUrl { get; set; }
+
+    /// <summary>
+    /// The Kernel instance used for the operation run.
+    /// </summary>
+    public Kernel? Kernel { get; set; }
```

Comment:
Should those properties be inits?

Do we have scenarios where the options needs to be setted after creation?

```suggestion
    public Kernel? Kernel { get; init; }
```

Reply:
Changes follow the existing pattern so I've created a new task for this https://github.com/microsoft/semantic-kernel/issues/6819. The class is internal so we are not adding any public API surface with these changes.
@SergeyMenshykh can you look at this change in the context of the OpenAPI graduation.